tutorials/11_gazebo_scene_viewer_tutorial.md

```diff
@@ -70,3 +50,23 @@ Launch the example to visualize the objects:
 You can use the `Tab` button to change the rendering engine.
 
 ![](img/gazebo_scene_viewer2_demo.gif)
```

Comment:
One thing I noticed is that these images are being rendered too large, and the page is not horizontally scrollable. I think we should reduce their size.

Reply:
reized gifs to 50%. 8c6fb3c